README.md

```diff
@@ -1,85 +1,20 @@
-# <img src="https://cloud.githubusercontent.com/assets/7833470/10423298/ea833a68-7079-11e5-84f8-0a925ab96893.png" width="60"> Personal API - Weekend Lab
+This API is designed to allow users to look up technical information about different vehicles (engine type, engine size, number of doors, etc) and is located at https://quiet-depths-71345.herokuapp.com/
```

Comment:
You want to keep the `#` before headings so things display visually bold and large and look like a heading instead of plain text.